python/grass/script/tests/test_script_task.py

```diff
@@ -0,0 +1,12 @@
+import pytest
```

Comment:
```suggestion
```

Seems like flake8 doesn't like that I didn't use any fixtures at all finally, so pytest is unused. Would this work?